### ISSUE TITLE
Cannot able to navigate through from one level to another level in the tree item present in left pane of workbook.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#391]
### Steps to Reproduce
1. Launch Xamarin Workbooks application. This will invokes "Create a new Workbook" window.
2. Create a Workbook with WPF Platform. Navigate to Help-> Tutorial -> Hello Workbooks and invoke it.
3. Navigate through Tree items from one level to another level (Parent Hello workbooks to Child Hello workbooks and vice versa , refer screenshots) in left pane using keyboard.

### Expected Behavior
Should be able to navigate through the tree item from one level to another (Parent Hello workbooks to Child Hello workbooks and vice versa)
### Actual Behavior
Cannot able to navigate through the tree item from one level to another (Parent Hello workbooks to Child Hello Workbooks and vice versa).
Keyboard focus remains at child Hello workbooks only (Please refer inspect screenshot below)
![screenshot 193](https://user-images.githubusercontent.com/22956130/40768423-e8790f4a-64d2-11e8-90ba-cd50aba7c8df.png)

### Screenshots:
![screenshot 192](https://user-images.githubusercontent.com/22956130/40768414-df7adf4a-64d2-11e8-93f6-bac50701e31d.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

### Log File

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.4.7

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
